internal/exec/stages/files/filesystemEntries.go

```diff
@@ -275,7 +275,7 @@ func (tmp fileEntry) create(l *log.Logger, u util.Util) error {
 	case regular && f.Contents.Source == nil:
 		break
 	default:
-		return fmt.Errorf("Ignition encountered an internal error processing %q and must die now. Please file a bug", f.Path)
+		return fmt.Errorf("ignition encountered an internal error processing %q and must die now. Please file a bug", f.Path)
```

Comment:
Ignition is a proper noun.  Let's rephrase the error or mute the staticcheck warning.